gammapy/estimators/utils.py

```diff
@@ -779,6 +782,61 @@ def get_rebinned_axis(fluxpoint, axis_name="energy", method=None, **kwargs):
     return axis_new
 
 
+def combine_significance_maps(maps):
+    """Computes excess and significance for a set of datasets.
+    The significance computation assumes that the model contains
+    one degree of freedom per valid energy bin in each dataset.
+    This method implemented here is valid under the assumption
+    that the TS in each independent bin follows a Chi2 distribution,
+    then the sum of the TS also follows a Chi2 distribution (with the sum of degree of freedom).
+
+    See, Zhen (2014): https://www.sciencedirect.com/science/article/abs/pii/S0167947313003204,
+    Lancaster (1961): https://onlinelibrary.wiley.com/doi/10.1111/j.1467-842X.1961.tb00058.x
+
+
+    Parameters
+    ----------
+    maps : list of `~gammapy.estimators.FluxMaps`
+        List of maps with the same geometry.
+
+    Returns
+    -------
+    results : dict
+        Dictionary with keys :
+        - "significance" : joint significance map.
+        - "df" : degree of freedom map (one norm per valid bin).
+        - "npred_excess" : summed excess map.
+        - "estimator_results" : dictionary containing the estimator results for each dataset.
+
+    See also
+    --------
+    get_combined_significance_maps : same method but computing the significance maps from estimators and datasets
+
+    """
+
+    geom = maps[0].ts.geom.to_image()
+    ts_sum = Map.from_geom(geom)
+    ts_sum_sign = Map.from_geom(geom)
+    npred_excess_sum = Map.from_geom(geom)
+    df = Map.from_geom(geom)
+
+    for result in maps:
+        df += np.sum(result["ts"].data > 0, axis=0)  # one dof (norm) per valid bin
+        ts_sum += result["ts"].reduce_over_axes()
+        ts_sum_sign += (
+            result["ts"] * np.sign(result["npred_excess"])
+        ).reduce_over_axes()
+        npred_excess_sum += result["npred_excess"].reduce_over_axes()
+
+    significance = Map.from_geom(geom)
+    significance.data = ts_to_sigma(ts_sum.data, df.data) * np.sign(ts_sum_sign)
+    return dict(
+        significance=significance,
+        df=df,
+        npred_excess=npred_excess_sum,
```

Comment:
The `estimator_results` is not present (coherence with the docstring)

Reply:
added